opentracing/standard/raw.py

```diff
@@ -30,13 +30,13 @@ def __init__(self, trace_context, operation_name):
         self.operation_name = operation_name
         self.start_time = None
         self.duration = None
-        self.tags = None  # array of (key,vale) pairs when initialized
+        self.tags = None  # dict when initialized
         self.logs = None  # array of RawLog's when initialized
 
-    def add_tag(self, key, value):
+    def set_tag(self, key, value):
         if self.tags is None:
-            self.tags = []
-        self.tags.append((key, value))
```

Comment:
It's tough to do an implementation of "undefined" behavior without, well, defining it -- which is I suppose the argument in favor of "skeleton"/"util" vs a "standard" tracer.  Append is the more common denominator, I've switched back to that, but I'm guessing this is a pretty short half-life file.

> we could have a super-simple polyglot standalone "OpenTracing" tracer service
@bensigelman I like that idea, was thinking about the same thing with the name "basic tracer" or similar.  Low on bells and whistles to emphasize core duties.